ReactNativeClient/lib/components/note-body-viewer.js

```diff
@@ -64,6 +69,8 @@ class NoteBodyViewer extends Component {
 			resources: this.props.noteResources,
 			codeTheme: theme.codeThemeCss,
 			postMessageSyntax: 'window.joplinPostMessage_',
+			enableLongPress: shim.isReactNative(),
+			longPressDelay: 500, // TODO use system value
```

Comment:
I know how to get system long press delay on Android but not sure if it's worth writing native code just to get that. Plus I'd need some help with iOS here.